### ISSUE TITLE
[Feature request] Remove outdated password requirements

### ISSUE THREAD

**Issue by NiklasBr** [pimcore/pimcore#5107]
# Feature Request
You've all [read this XKCD comic](https://xkcd.com/936/) by now, haven't you? `correctbatteryhorsestaple` and all that....

It is no longer recommended to use complex rules for user passwords, for example these which recommend against the current implementation:

- [Microsoft password guidance, anti-pattern #2](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/06/Microsoft_Password_Guidance-1.pdf)
- [NIST on complexity composition rules](https://pages.nist.gov/800-63-3/sp800-63b.html)
- [GCHQ guidance](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/458857/Password_guidance_-_simplifying_your_approach.pdf)

In summary: Just a minimum character length please.

**Comment by brusch** [comment]
+ do server-side validation as well

**Comment by ondrejd** [comment]
I'm looking into sources of 6.4.2 and doesn't see these changes.
